manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0

```

```suggestion
PackageVersion: 1.3.0
```
Must match Apps&Features / Control Panel